### DISCUSSION TITLE
A32NX v0.11.1

### DISCUSSION THREAD

**Discussion by Jm62223** [flybywiresim/aircraft#8417] [General]
Hello, since the new version of this excellent aircraft, I can't find a solution for putting a visible co-pilot in the cockpit.
Do you have a solution?
Thank you

**Comment by tracernz** [comment]
Try the pilot avatar toggles https://docs.flybywiresim.com/fbw-a32nx/feature-guides/flypados3/settings/#realism

**Comment by Jm62223** [comment]
‌Thank you very much for this information.
My co-pilot is back.
 

De : "Michael Corcoran"
A : "flybywiresim/aircraft" ,"Jm62223" ,"Author"
Envoyé: mercredi 17 Janvier 2024 03:52
Objet : Re: [flybywiresim/aircraft] A32NX v0.11.1 (Discussion #8417)
 

 

Try the pilot avatar toggles https://docs.flybywiresim.com/fbw-a32nx/feature-guides/flypados3/settings/#realism
